pennylane_qiskit/qiskit_device.py

```diff
@@ -234,6 +234,8 @@ def apply_operations(self, operations):
 
             if operation.endswith(".inv"):
                 gate = gate.inverse()
+                if "QubitUnitary" in operation:
+                    qregs = list(reversed(qregs))
```

Comment:
Good catch! :blush: :100: Could this be merged with the logic on lines 227-230? As is now, there would be two reversals in certain cases. Also, could it be a problem for `"QubitStateVector"` too?

Reply:
Yes I considered this! I thought the same as you as well; that `QubitStateVector` shouldn't be invertible 😆